dir.targets

```diff
@@ -32,7 +32,7 @@
 
   <!-- Respect the traits set up by PlatformSpecific attribute. -->
   <PropertyGroup>
-    <XunitOptions Condition="'$(OS)'=='Windows_NT'">$(XunitOptions) -notrait category=nonwindowstests</XunitOptions>
+    <XunitOptions Condition="'$(OS)'=='Windows_NT'">$(XunitOptions) -notrait category=nonwindowstests -notrait category=mono-windows-failing</XunitOptions>
```

Comment:
Should this be conditional on `MonoBuild==true` analagously to `mono-osx-failing` below?